BLOCKFILE

```diff
@@ -49,6 +49,7 @@ hw/ip/kmac/rtl/*
 hw/ip/lc_ctrl/rtl/*
 hw/ip/otbn/rtl/*
 hw/ip/otp_ctrl/rtl/*
+hw/ip/otp_macro/rtl/*
```

Comment:
We probably should also add `hw/ip/otp_macro/data/otp_macro.hjson` below, right?

Reply:
I add the hjson file in my PR https://github.com/lowRISC/opentitan/pull/27132, which I would really appreciate if any of you can review.